<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>338</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>748</width>
    <height>338</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>748</width>
    <height>338</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lb_icon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>338</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/tb/mimi.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>394</x>
     <y>19</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tb_login">
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QLineEdit" name="le_tel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>13512345678</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>72</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_pwd">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>72</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_pwd">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_commit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_register">
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QLineEdit" name="le_pwd_register">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>输入长度不超过20位</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tel_register">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>72</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_commit_register">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tel_register">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>请输入11位数字</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_pwd_register">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>72</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear_register">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_confirm_register">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_confirm_register">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>72</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_name_register">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>输入长度不超过10位</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_name_register">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>72</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>昵称</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_tel_register</tabstop>
  <tabstop>le_pwd_register</tabstop>
  <tabstop>le_confirm_register</tabstop>
  <tabstop>le_name_register</tabstop>
  <tabstop>pb_commit_register</tabstop>
  <tabstop>pb_clear_register</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>le_tel</tabstop>
  <tabstop>le_pwd</tabstop>
  <tabstop>pb_commit</tabstop>
  <tabstop>pb_clear</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
